OpenKeychain/src/main/java/org/sufficientlysecure/keychain/service/PassphraseCacheService.java

```diff
@@ -314,7 +314,8 @@ public void onReceive(Context context, Intent intent) {
     private static PendingIntent buildIntent(Context context, long keyId) {
         Intent intent = new Intent(BROADCAST_ACTION_PASSPHRASE_CACHE_SERVICE);
         intent.putExtra(EXTRA_KEY_ID, keyId);
-        return PendingIntent.getBroadcast(context, REQUEST_ID, intent,
+        Long _id = System.currentTimeMillis();
```

Comment:
please use ``long`` instead of ``Long`` and follow the code style guidelines (this means naming the var ``requestCode`` instead of ``_id``
Then also add a comment to the code indicating why this request id needs to be unique.